classes/03class/exercises/c03-docker06/chisholm-ray/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:latest
+ENV YOUR_NAME="null"
+EXEC echo "Hello $YOUR_NAME"
```

Comment:
This threw an error for me at the build time -> `docker build -t hello-name:v1 .`
`Error response from daemon: dockerfile parse error line 7: unknown instruction: EXEC`
I usually use CMD or ENTRYPOINT when I need to pass arguments, however it is worth double checking if this builds correctly for you.